rfc/src/rfcs/0010-quantifiers.md

```diff
@@ -0,0 +1,207 @@
+- **Feature Name:** Quantifiers
+- **Feature Request Issue:** [#2546](https://github.com/model-checking/kani/issues/2546) and [#836](https://github.com/model-checking/kani/issues/836)
+- **RFC PR:** [#](https://github.com/model-checking/kani/pull/)
+- **Status:** Unstable
+- **Version:** 1
+
+-------------------
+
+## Summary
+
+Quantifiers are like logic-level loops and a powerful reasoning helper, which allow us to express statements about objects in a domain that satisfy a given condition. There are two primary quantifiers: the existential quantifier (∃) and the universal quantifier (∀).
+
+1. The existential quantifier (∃): represents the statement "there exists." We use to express that there is at least one object in the domain that satisfies a given condition. For example, "∃x P(x)" means "there exists an x such that P(x) is true."
+
+2. The universal quantifier (∀): represents the statement "for all" or "for every." We use it to express that a given condition is true for every object in the domain. For example, "∀x P(x)" means "for every x, P(x) is true."
+
+## User Impact
+
+Quantifiers enhance users' expressive capabilities, enabling the verification of more intricate properties in a concise manner.
```

Comment:
Is it actually try that expressive capabilities are enhanced (I'm not even sure that statement is well defined). In a Turing-complete language expressiveness is not going to change through quantifiers.

Reply:
I wrote that phrase with "enabling developers to write more general properties" or something with that sort in mind. Removing it for clarity.